test/hotspot/gtest/logging/logTestFixture.cpp

```diff
@@ -96,7 +96,15 @@ void LogTestFixture::restore_config() {
     *str++ = '\0';
 
     char* decorators = str;
+    str = strchr(str, ' ');
+    if (str != NULL) {
+      *str = '\0';
+    }
 
+// TODO: Restore following code if options for existing log output can be changed.
```

Comment:
I think this change needs a better explanation and should be done in a separate RFE. I've file https://bugs.openjdk.java.net/browse/JDK-8273610 and will post a PR soon.

Reply:
I posted the PR: https://github.com/openjdk/jdk/pull/5472

Reply:
Thanks! I will modify this PR after merging #5472 .

Reply:
I merged #5472 .